packages/headless/src/api/search/search/product-recommendation.ts

```diff
@@ -1,4 +1,12 @@
 export interface ProductRecommendation {
+  /**
+   * Document Uri in the index. Still useful for analytics.
```

Comment:
I think the `Still useful for analytics.` is not strictly necessary.

Reply:
I think so as well 🤔 
If it's necessary, I'd suggest:
```suggestion
   * Document URI in the index. Useful for analytics.
```

Reply:
Good suggestion. thanks.